docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+function do_help {
+  echo HELP:
+  echo "Supported commands:"
+  echo "   master              - Start a Kudu Master"
+  echo "   tserver             - Start a Kudu TServer"
+  echo "   help                - print useful information and exit"
+  echo ""
+  echo "Other commands can be specified to run shell commands."
+
+  exit 0
+}
+
+if [ "$1" = 'master' ]; then
+	exec kudu-master -fs_wal_dir /data/kudu-master --logtostderr \
+  --use_hybrid_clock=false
+elif [ "$1" = 'tserver' ]; then
+  exec kudu-tserver -fs_wal_dir /data/kudu-tserver \
+  -tserver_master_addrs ${KUDU_MASTER} \
+  --logtostderr --use_hybrid_clock=false
+elif [ "$1" = 'help' ]; then
+  do_help
+fi
+
```

Comment:
Since the ENTRYPOINT has been changed, it is necessary that the COMMAND argument to " docker run" became an argument to ENTRYPOINT :
https://docs.docker.com/reference/run/#cmd-default-command-or-options